CONTRIBUTING.md

```diff
@@ -34,9 +37,25 @@ Once your PR is open, you can ask for review by a maintainer of your platform. W
 is that a PR must be approved by at least two maintainers of winit before being merged, including
 at least a maintainer of the platform (a maintainer making a PR themselves counts as approving it).
 
+Once your PR is deemed ready, the merging maintainer will take care of resolving conflicts in
+`CHANGELOG.md` (but you must resolve other conflicts yourself). Doing this requires that you check the
+"give contributors write access to the branch" checkbox when creating the PR.
+
 ## Maintainers & Testers
 
 The current [list of testers and contributors](https://github.com/rust-windowing/winit/wiki/Testers-and-Contributors)
 can be found on the Wiki.
 
 If you are interested in contributing or testing on a platform, please add yourself to that table!
+
+## Making a new release
+
+If you believe a new release is warranted, you can make a pull-request with:
+- An updated version number (remember to change the version everywhere it is used).
+- A new section in the changelog (below the `# Unreleased` section).
+
+This gives contributors an opportunity to squeeze in an extra PR or two that they feel is valuable
+enough to warrant blocking the release a little.
+
+Once the PR is merged, a CI job will automatically release the new version. Remember that the release
+date in the changelog must be kept in check with the actual release date.
```

Comment:
If we want to be pedantic, this is not true. A maintainer has to create a tag in order for the CI to make the release. The easiest way to do this is to make a new release from the GitHub Releases page and create a tag from there.

Reply:
Heh, I would not say that's "being pedantic"; that's just being correct 😉 (I thought it worked the way I wrote it). I've updated the text now.

Reply:
In that's case it's nice to clear that up. 😁